### ISSUE TITLE
Suggestion to rename bootstrap chart

### ISSUE THREAD

**Issue by v1k0d3n** [att-comdev/openstack-helm#96]
@alanmeadows i'm a bit torn on this idea, because it will require some work, but I think during the 0.2.0 release we should rename the `bootstrap` chart to something that is a bit more clearer for new users (even thought it's lightly documented). so we can either make the documentation more clear, rename, or both. thoughts?

**Comment by larryrensing** [comment]
i agree, if it's only being used for ceph pvc support it should be renamed to something that reflects the intent of the chart

**Comment by intlabs** [comment]
Renaming to something like 'ceph-pvc-volumeclass' makes sense to me.

**Comment by v1k0d3n** [comment]
@intlabs exactly along the lines of what i'm suggesting. i was thinking about `ceph-storageclass-boostrap` or something similar honestly.
